minerals/xyz.armcord.ArmCord.metainfo.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
Has this file been submitted upstream?

Reply:
I'd rather keep the metainfo downstream, so that I can make sure it's constantly up-to-date without relying on .patch files. From my experience with upstream developers, unless they actually use that metainfo.xml file, they won't put in effort to update it, and even then it's usually only the bare minimum, not even release notes are added.